common/bpf/map.go

```diff
@@ -176,6 +177,10 @@ func OpenMap(path string) (*Map, error) {
 }
 
 func (m *Map) OpenOrCreate() (bool, error) {
+	if m.isOpen {
```

Comment:
Is this boolean flag absolutely required or can the open state be computed solely from `m.fd != 0`?